internal/plugins/helm/v1/scaffolds/templates/makefile.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package templates
+
+import (
+	"sigs.k8s.io/kubebuilder/pkg/model/file"
+)
+
+var _ file.Template = &Makefile{}
+
+// Makefile scaffolds the Makefile
+type Makefile struct {
+	file.TemplateMixin
+
+	// Image is controller manager image name
+	Image string
+	//Kustomize version to use in the project
+	KustomizeVersion string
+}
+
+// SetTemplateDefaults implements input.Template
+func (f *Makefile) SetTemplateDefaults() error {
+	if f.Path == "" {
+		f.Path = "Makefile"
+	}
+
+	f.TemplateBody = makefileTemplate
+
+	f.IfExistsAction = file.Error
+
+	if f.Image == "" {
+		f.Image = "controller:latest"
+	}
+
+	return nil
+}
+
+//nolint:lll
+const makefileTemplate = `
+# Image URL to use all building/pushing image targets
+IMG ?= {{ .Image }}
+
+all: docker-build docker-push install deploy
```

Comment:
```suggestion
all: docker-build 
```
I need to update that